airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -135,6 +147,7 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
         // When a scope is cancelled temporal will throw a CanceledFailure as you can see here:
         // https://github.com/temporalio/sdk-java/blob/master/temporal-sdk/src/main/java/io/temporal/workflow/CancellationScope.java#L72
         // The naming is very misleading, it is not a failure but the expected behavior...
+        recordFailureCountMetric(connectionUpdaterInput, FailureCause.CANCELED);
```

Comment:
I'm curious when did cancel would ever be triggered since I'd think any cancel is caught by the sync child workflow. @benmoriceau would you know?